### PR TITLE
Update 1.9.4

### PR DESCRIPTION
**Pull request by SC426**
Removed 2 old patches:
https://github.com/lz4/lz4/pull/962/files
https://github.com/lz4/lz4/pull/972/files

Jira ticket: https://anaconda.atlassian.net/browse/PKG-175
Upstream repo: https://github.com/lz4/lz4
Changelog: https://github.com/lz4/lz4/blob/dev/NEWS

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,15 +9,11 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 030644df4611007ff7dc962d981f390361e6c97a34e5cbc393ddfbe019ffe2c1
+  sha256: 0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b
   patches:
```
Comment (ELundby45) [inline recipe/meta.yaml]: Nit: `patches:` could be removed

**Review by ELundby45 — APPROVED** [review]

**Review by cbouss — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,15 +9,10 @@ package:
...
-    # See: https://github.com/lz4/lz4/pull/972
-    - patches/CVE-2021-3520.patch
+  sha256: 0b0e3aa07c8c063ddf40b082bdf7e37a1562bda40a0ff5272957f3e987e0e54b
 
 build:
-  number: 1
+  number: 0
 
 requirements:
   build:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:18]: `patch  # [unix]` and `m2-patch  # [win]` can be removed as we do not apply any patches

File: recipe/meta.yaml [hunk 3]
```diff
@@ -75,12 +70,12 @@ outputs:
         - test -f ${PREFIX}/lib/liblz4.a      # [unix]
         - if not exist %LIBRARY_LIB%\\liblz4_static.lib exit 1  # [win]
 
-
 about:
   home: https://lz4.github.io/lz4/
   license: BSD-2-Clause
   license_family: BSD
   license_file: lib/LICENSE
+  license_family: BSD
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `license_family` is a duplication of https://github.com/AnacondaRecipes/lz4-c-feedstock/pull/3/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR76

**Pull request merged**